### ISSUE TITLE
Different batch size and Learning Rate decay

### ISSUE THREAD

**Issue by Dheeraj-B** [Dheeraj-B/ProjectSeg#1]
Need to experiment with this. The base inception architecture takes batches of size 32. Need to try fitting with sizes of 64, 128 and so on. Also need to figure out how to plot the training loss progression using keras. This will help in designing learning rate schedule across epochs.
